src/main/java/roomescape/Controller/TimeController.java

```diff
@@ -0,0 +1,34 @@
+package roomescape.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.*;
+import roomescape.Service.TimeService;
+import roomescape.Domain.Time;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/times")
+public class TimeController {
+    @Autowired
+    private TimeService timeService;
+    @GetMapping
+    public List<Time> getAllTime(){
+        return timeService.getAllTime();
+    }
```

Comment:
한 줄 띄우면 좋을 거 같네용
그리고 아래에서 준호님께서 말씀해주신대로 라인 포맷팅으로 깔끔하게 코드 작성할 수 있답니당👀 
```suggestion
    @Autowired
    private TimeService timeService;
    
    @GetMapping
    public List<Time> getAllTime() {
        return timeService.getAllTime();
    }
```

Reply:
넵 수정하겠습니다!!